code/modules/modular_computers/NTNet/NTNRC/conversation.dm

```diff
@@ -1,24 +1,20 @@
-var/global/ntnrc_uid = 0
-
 /datum/ntnet_conversation/
 	var/id = null
 	var/title = "Untitled Conversation"
 	var/datum/computer_file/program/chatclient/operator // "Administrator" of this channel. Creator starts as channel's operator,
 	var/list/messages = list()
 	var/list/clients = list()
 	var/password
-	var/source_z
+	var/datum/computer_network/network
```

Comment:
Would this be a better weakref?

Reply:
yeah, networks can get deleted after all

Reply:
actually made networks qdel their convos when deleted so should be okay now, no lingering references left